CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 1.29.5 - 31 March 2025
+### Added
+* The new [Docker DX](https://marketplace.visualstudio.com/items?itemName=docker.docker) extension by Docker Inc. will be installed alongside this extension.
```

Comment:
@fiveisprime should we mention it's ok to remove if they want?

Reply:
"...will be installed alongside this extension **as an optional dependency."**